source/fuzz/transformation_split_block.cpp

```diff
@@ -40,31 +40,28 @@ bool TransformationSplitBlock::IsApplicable(
     // We require the id for the new block to be unused.
     return false;
   }
-  auto base_instruction =
-      context->get_def_use_mgr()->GetDef(message_.base_instruction_id());
-  if (!base_instruction) {
+  auto instruction_to_split_before =
+      FindInstruction(message_.instruction_to_split_before(), context);
```

Comment:
Perhaps `FindInstruction` should return an instruction iterator for maximum flexibility.

Reply:
Unfortunately this won't work because there may not be a basic block associated with the instruction (if the instruction does not exist).